packages/react-ui/.styleguide/helpers.js

```diff
@@ -22,13 +22,22 @@ const excludedComponents = [
   'SidePageContext',
 ];
 
-const sectionComponents = ['Modal', 'SidePage'];
+const sectionComponents = ['Modal', 'SidePage', 'Tabs'];
 
 const findComponentsInSection = (dirPath, name) => {
-  const reg = new RegExp(`${name}[a-zA-Z]*\.tsx`);
+  // Looks for entries like `ParentCompChildComp.tsx`
+  // In `ModalClose` the word `Modal` represents parent component and the word `Close` child component
+  const surnamePattern = new RegExp(`${name}[a-zA-Z]*\.tsx`);
+  // Looks for entries like `Parents.tsx`
+  // In `Tabs` the word `Tabs` represents parent component and the word `Tab` child component
+  const substringPattern = name.slice(0, -1) + '.tsx';
```

Comment:
Это довольно Таба-специфичная логика и будет работать в одном единственном случае.
Предлагаю заменить на более явную реализацию, чтобы не делать лишних вычислений.

Например так:
```suggestion
  const substringPatternTab = 'Tab.tsx';
```

<br/>

Или обновить регулярку выше:
```js
const surnamePattern = new RegExp(`(?:${name}[a-zA-Z]*|Tab)\.tsx`);
```